.gitignore

```diff
@@ -3,3 +3,4 @@
 /build
 /*/build
 /*/out
+/application-profile/collapse-single-ap-output.ap.json
```

Comment:
hmm. yes. I think this one is produced by one of tests. should we not check if it can be avoided.

Reply:
so in `CollapseApplicationProfileTest`

let's remove 
```
   //    JsonNode jsonNode = applicationProfileConfig.getSerializer().apply(collapse);
    JsonNode jsonNode = new ApplicationProfileSimpleJson().apply(collapse);

    print(jsonNode);
```

and also 
```
private void print(JsonNode jsonNode) {
    try {
      ObjectMapper mapper = new ObjectMapper();
      ObjectWriter writer = mapper.writer(new DefaultPrettyPrinter());

      File file = new FileSystemResource("collapse-single-ap-output.ap.json").getFile();
      writer.writeValue(file, jsonNode);
    }
    catch (IOException e) {
      throw new RuntimeException(e);
    }
  }
```